sdk/Lusid.Sdk.Tests/tutorials/Valuations/InlineBondValuation.cs

```diff
@@ -0,0 +1,89 @@
+            var flowConventions = new FlowConventions(
+                scope: null,
+                code: null,
+                currency: "GBP",
+                paymentFrequency: "6M",
+                rollConvention: FlowConventions.RollConventionEnum.MF,
+                dayCountConvention: FlowConventions.DayCountConventionEnum.Act365,
+                holidayCalendars: new List<string>(),
+                settleDays: 2,
+                resetDays: 2
+            );
+            
+            // Use the GetValuationOfWeightedInstruments to price our Bond
+            // We price the Bond for a date between the Bond's startDate and maturityDate
+
+            ListAggregationResponse bondValuationResult = _apiFactory.Api<IAggregationApi>().GetValuationOfWeightedInstruments(
+                new InlineValuationRequest(recipeId: new ResourceId(scope: "MyRecipeScope", code: "default"),
+                    valuationSchedule: new ValuationSchedule(
+                        effectiveFrom: new DateTimeOrCutLabel("2020-09-01"),
+                        effectiveAt: new DateTimeOrCutLabel("2020-09-01"),
+                        tenor: "1D"
+                    ),
+                    
+                    metrics: new List<AggregateSpec>
+                {
+                    new AggregateSpec(key: "Instrument/default/Name", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec(key: "Holding/default/PV", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec(key: "Instrument/OTC/Bond/CashFlows", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec(key: "Instrument/OTC/Bond/AccruedInterest", AggregateSpec.OpEnum.Value)
+                }, 
+                    
+                    instruments: new List<WeightedInstrument>
+                {
+
+                    new WeightedInstrument(
+                        quantity: 1,
+                        holdingIdentifier: "test-inline-val",
+                        instrument: new Bond(
+                            startDate: new DateTimeOffset(2020, 1, 1, 0, 0, 0,TimeSpan.Zero),
+                            maturityDate: new DateTimeOffset(2021, 1, 1, 0, 0, 0,TimeSpan.Zero),
+                            domCcy: "GBP",
+                            flowConventions: flowConventions,
+                            principal: 10000,
+                            couponRate: new decimal(0.05),
+                            identifiers: new Dictionary<string, string>(),
+                            instrumentType: LusidInstrument.InstrumentTypeEnum.Bond
+                        )
+                        )
+                }));
+            
+            // Test that the valuation engine returns a valid result
+            Assert.That(bondValuationResult.Data, Has.Count.EqualTo(1));
+            
+            // Test that the PV calculation is what we expect
+            Assert.That(bondValuationResult.Data[0]["Holding/default/PV"], Is.EqualTo(10336.986301369863));
```

Comment:
What data is used in the calculation of the present value of the bond? Will this value change over time as the maturity date is hard coded? As interest rates change?

Reply:
I will send you excel with the calculation. The value will change if the discount curve changes, but we have hard coded that into the test.

Reply:
@MikeMcGarry, you can check out this test https://github.com/finbourne/lusid-sdk-csharp-preview/blob/e6770cd55906e0f2f362159b0d1ba7b4f6e9eaf2/sdk/Lusid.Sdk.Tests/tutorials/Ibor/Valuation.cs#L148

where I did something similar to @slemasne, there it values the bond across a week and check that every day (effectiveAt), the PV of the bond varies.